src/makeCalculator.js

```diff
@@ -36,8 +36,49 @@
  *
  * @return {object}
  */
-function makeCalculator() {
-  // write code here
+function makecalculator() {
+  const calc = {
+
+    result: 0,
+
+    operate(callback, value = 0) {
+      callback(value);
+
+      return calc;
+    },
+
+    add(value = 0) {
+      calc.result += value;
+
+      return calc;
```

Comment:
```suggestion
```
It's redundant here. As task description says:

> - The `operate` and `reset` methods can be called in a chain.

So it means that only `reset` and `operate` should return the calculator to have the possibility to be used in the chain.
But ok